service/src/main/kotlin/fi/tampere/trevaka/TrevakaProperties.kt

```diff
@@ -27,7 +27,7 @@ data class InvoiceProperties(
     val salesOrganisation: String = "1312",
     val distributionChannel: String = "00",
     val division: String = "00",
-    val salesOrderType: String = "Z001",
+    val salesOrderType: String = "ZPH",
```

Comment:
Need to rethink this, when credit memos (hyvityslasku) are handled. "ZHLP" is used for them.